### ISSUE TITLE
Separating pages Coaching and Topics 

### ISSUE THREAD

**Issue by DavidJanku** [ea-czech-republic/efektivnialtruismus.cz#124]
split page Topics & Coaching (currently http://effectivethesis.com/theses/) to two pages: one page called Topics and the other called Coaching

Coaching would have URL http://effectivethesis.com/thesis-coaching/ and Topics page would have URL http://effectivethesis.com/theses/

Here is how the Topics page should look like:
![topics_page](https://user-images.githubusercontent.com/31438579/64474872-bedad480-d172-11e9-820a-d0a146e952dd.jpg)

**Comment by hnykda** [comment]
@DavidJanku And how would the coaching page look like?

**Comment by hnykda** [comment]
So I did the following:

### Thesis
* functionality is the same (filtering on thesis).
* the form is not added yet, I just linked to contact us - let's do it in a separate ticket maybe (as it doesn't seem to be critical for the demo purposes)?
* all fields are editable in admin
![image](https://user-images.githubusercontent.com/2741256/66251326-39394d00-e74f-11e9-8cd9-7d0ae20d0fb4.png)



### Coaching
* I let in "How it works" and the form from thesis
![image](https://user-images.githubusercontent.com/2741256/66251330-46eed280-e74f-11e9-9530-8a12f3d8ae1d.png)

**Comment by hnykda** [comment]
Deployed on beta

**Comment by DavidJanku** [comment]
> @DavidJanku And how would the coaching page look like?

the same as it looks in the current version, just without the tags. What you proposed [here](https://beta.effectivethesis.com/theses/) is good, but please add the editable text at the top (as it is in the [current version  ](http://effectivethesis.com/theses/)

Re- Topics page  
- please substitute the current picture at the top of the page with [this one](https://www.pexels.com/photo/ballpen-blank-desk-journal-606541/), so we don't have same pictures on Topics and Coaching pages. Also, please substitute the text "Use your thesis to improve the world. Our coaching will help you." on the picture with "Stir your inspiration and make your research useful" as proposed in the original draft above.
 - please add the text "got interested in any of these topics? Please let us know! we can....." under the tags and the short form as proposed in the original draft above. Ideally, the text would be editable in admin. 

Once this is done, feel free to deploy it

**Comment by hnykda** [comment]
> the same as it looks in the current version, just without the tags. What you proposed here is good, but please add the editable text at the top (as it is in the current version
OK, like this?
![image](https://user-images.githubusercontent.com/2741256/66260992-6f65e380-e7c6-11e9-94f2-afeb799e5b01.png)

> please substitute the current picture at the top of the page with this one, so we don't have same pictures on Topics and Coaching pages
done

> Stir your inspiration and make your research useful
Done

> please add the text "got interested in any of these topics? Please let us know! we can....." under the tags and the short form as proposed in the original draft above. Ideally, the text would be editable in admin.
The text is editable in admin (see in the example, there is that "I am editable in admin" or something, that's my "marker" for things in admin). Either send me the whole text please or I can just deploy it and you can put it there.

**Comment by DavidJanku** [comment]
Great! One more thing I realised - it would be good if the text field at the beginning of the Topics page would be expandable - ie there would be first 3 sentences and then "Read more" button/hyperlinked text or something like that. This is because the text I'm planning to place there is pretty long - I've updated it and you can see it [here](https://beta.effectivethesis.com/theses/).  

Also, it would be good to finish this part of the Topics page 
![topics_page_2](https://user-images.githubusercontent.com/31438579/66261298-f321cf00-e7ca-11e9-94fa-ace83e1da5ad.jpg)

**Comment by hnykda** [comment]
@rakyi would you be able to take a look at it and try to make it expandable (I didn't know what David meant before by that, now I imagine some [collapsable](https://getbootstrap.com/docs/4.3/components/collapse/#example))

Adding that additional part. You can edit that text in admin - feel free to expand it, it's what I wrote in the previous message:
>The text is editable in admin (see in the example, there is that "I am editable in admin" or something, that's my "marker" for things in admin). I deployed that, you can now edit it in admin. I just let a phrase "Got interested in any of these topics? Let us know!" in.

Adding the contact form is a bit harder, not sure if Martin would be able to do that now (either as HTML in admin or in the codebase). I recommend just add a link to the contact page or something for now.

**Comment by DavidJanku** [comment]
Great, thanks! Maybe to specify it a bit more - there are two pieces of text above the tags section - one is starting by "How to choose a research topic. As you arrive at this website...." and finishes with "Dive in and get inspired!" - this one is the long text that should be collapsable. under this piece of text, there is another piece starting by "If you get interested in any of these topics, let us know. We can...." - this one should not be collapsable.

**Comment by rakyi** [comment]
Text starting with "If you get interested in any of these topics..." is there twice now, both above and below the tags. That is probably not right.

So you want two admin editable sections above the tags: first one collapsible and second one static, is that correct?

**Comment by DavidJanku** [comment]
> Text starting with "If you get interested in any of these topics..." is there twice now, both above and below the tags. That is probably not right.

Yeah, it doesn't look good - I will probably change the text of the field below the tags, but please keep the field there and keep it editable in admin,


> So you want two admin editable sections above the tags: first one collapsible and second one static, is that correct?

Yes, correct.

**Comment by rakyi** [comment]
I pushed the changes to beta, please check.

**Comment by DavidJanku** [comment]
> I pushed the changes to beta, please check.

works great, thanks! I would just change the hyperlinked text from "+ Show more" to "Read more"
